MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -198,94 +198,115 @@
+                                                    <constraint firstItem="u1L-N9-mMS" firstAttribute="top" secondItem="c3B-6D-Ukh" secondAttribute="top" id="Az5-l0-uTr"/>
+                                                    <constraint firstAttribute="trailing" secondItem="BR8-8N-Rhk" secondAttribute="trailing" id="Gpb-6j-pF2"/>
+                                                    <constraint firstItem="u1L-N9-mMS" firstAttribute="leading" secondItem="c3B-6D-Ukh" secondAttribute="leading" id="YYj-Tj-cgd"/>
+                                                    <constraint firstItem="BR8-8N-Rhk" firstAttribute="top" secondItem="u1L-N9-mMS" secondAttribute="bottom" id="guF-uE-UOT"/>
+                                                    <constraint firstAttribute="trailing" secondItem="u1L-N9-mMS" secondAttribute="trailing" id="vXp-kb-y24"/>
+                                                </constraints>
+                                            </stackView>
+                                            <imageView userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="gkj-hG-NxS">
+                                                <rect key="frame" x="237" y="19" width="32" height="32"/>
+                                                <constraints>
+                                                    <constraint firstAttribute="height" constant="32" id="0ei-is-yQS"/>
+                                                    <constraint firstAttribute="width" constant="32" id="4qd-KG-vi7"/>
+                                                </constraints>
+                                            </imageView>
+                                        </subviews>
                                         <constraints>
-                                            <constraint firstAttribute="height" constant="32" id="0ei-is-yQS"/>
-                                            <constraint firstAttribute="width" constant="32" id="4qd-KG-vi7"/>
+                                            <constraint firstItem="gkj-hG-NxS" firstAttribute="top" secondItem="dhM-Ng-gx5" secondAttribute="top" constant="19" id="AOc-29-8va"/>
+                                            <constraint firstItem="c3B-6D-Ukh" firstAttribute="leading" secondItem="dhM-Ng-gx5" secondAttribute="leading" id="Dcy-eQ-VCk"/>
+                                            <constraint firstAttribute="bottom" secondItem="gkj-hG-NxS" secondAttribute="bottom" constant="19" id="EBu-yb-gc6"/>
+                                            <constraint firstAttribute="bottom" secondItem="c3B-6D-Ukh" secondAttribute="bottom" id="Ofy-g6-EBl"/>
+                                            <constraint firstItem="gkj-hG-NxS" firstAttribute="leading" secondItem="c3B-6D-Ukh" secondAttribute="trailing" id="mWA-J9-gaq"/>
```

Comment:
There needs to be a gap between the right edge of the street name label (and distance label) and the left edge of the shield view. The gap should be the same width as the right margin of the shield view and ideally the same width as the gap between the maneuver icon and the street name label.